src/Files.App/Data/Models/ItemViewModel.cs

```diff
@@ -970,18 +969,23 @@ private async Task LoadItemThumbnailAsync(ListedItem item, uint thumbnailSize =
 				{
 					await dispatcherQueue.EnqueueOrInvokeAsync(async () =>
 					{
+						// Assign the thumbnail/icon to the listed item
 						item.FileImage = await iconInfo.IconData.ToBitmapAsync();
-						if (!string.IsNullOrEmpty(item.FileExtension) &&
-							!item.IsShortcut && !item.IsExecutable &&
```

Comment:
Should `!item.IsShortcut && !item.IsExecutable` be deleted?

Reply:
I'm not sure why there were excluded in the first place.

Reply:
Shortcuts and executables have their own icon, so if we include them, the default icon will be an icon for a specific shortcut or executable.

Reply:
I'll add it back